cloud/aci-compose-features.md

```diff
@@ -0,0 +1,241 @@
+---
```

Comment:
Looks like this page is not added to the TOC currently

Reply:
I only linked these from the aci-integration and ecs-integration pages, I think it's all right, but I can change that if yo prefer.

Reply:
After Ben comment, added pages to TOC